ros_buildfarm/templates/snippet/from_base_image.Dockerfile.em

```diff
@@ -1,3 +1,4 @@
+@[if base_image]@
```

Comment:
Is this logic reversed? This looks like a new tag which won't necessarily be set and the old behavior will not persist. And the else uses base_name and base_tag_name which are also new.